File: ideal-ui/src/main/java/com/adyen/checkout/ideal/IdealSpinnerView.java
/*
 * Copyright (c) 2019 Adyen N.V.
 *
 * This file is open source and available under the MIT license. See the LICENSE file for more info.
 *
 * Created by arman on 20/2/2019.
 */

package com.adyen.checkout.ideal;

import android.content.Context;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import android.util.AttributeSet;

import com.adyen.checkout.base.model.payments.request.IdealPaymentMethod;
import com.adyen.checkout.issuerlist.IssuerListSpinnerView;

public final class IdealSpinnerView extends IssuerListSpinnerView<IdealPaymentMethod, IdealComponent> {

    public IdealSpinnerView(@NonNull Context context) {
        super(context);
    }

    public IdealSpinnerView(@NonNull Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public IdealSpinnerView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }
}

File: ideal/src/main/java/com/adyen/checkout/ideal/IdealSpinnerView.kt
/*
 * Copyright (c) 2019 Adyen N.V.
 *
 * This file is open source and available under the MIT license. See the LICENSE file for more info.
 *
 * Created by arman on 20/2/2019.
 */
package com.adyen.checkout.ideal

import android.content.Context
import android.util.AttributeSet
import com.adyen.checkout.components.model.payments.request.IdealPaymentMethod
import com.adyen.checkout.issuerlist.IssuerListSpinnerView

class IdealSpinnerView : IssuerListSpinnerView<IdealPaymentMethod, IdealComponent> {
    constructor(context: Context) : super(context)
    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)
    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr)
}
